src/controller/python/chip/yaml/runner.py

```diff
@@ -253,8 +278,12 @@ def __init__(self, test_step):
         args = test_step.arguments['values']
         request_data_as_dict = Converter.convert_list_of_name_value_pair_to_dict(args)
 
+        # When expireExistingSession is not provided it is true be default. This is because
+        # expireExistingSession argument was added later on to test a specific CASE situation
+        # where that specific test case did not want existing session to be expired. For more
+        # information see PR #20820.
+        self._expire_existing_session = request_data_as_dict.get('expireExistingSession', True)
```

Comment:
I think the key part here is that we are preserving the old behavior, and that behavior exists for a reason, and that reason is documented where we currently implement the behavior.

We should copy _that_ comment, explaining the reason, to here, to make it clear why the behavior is what it is.  So replace the comments with some version of https://github.com/project-chip/connectedhomeip/blob/35bd94da455fbab2bf0cc474f220a44c989d32d2/examples/chip-tool/commands/tests/TestCommand.cpp#L40-L42